src/common/components/ui/formik/RadioField.tsx

```diff
@@ -16,7 +16,7 @@ const RadioField = ({
     value,
     type: 'radio',
   });
-  const fieldId = getFieldId(name, 'radio', field.value);
+  const fieldId = getFieldId(name, 'radio', field.value as string | undefined);
```

Comment:
Please remember to reduce the number of warnings in `package.json` when you fix some Eslint errors.

Reply:
Done! 240 warnings